dist/info.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Info</title>
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="/info.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aleo:ital,wght@0,400;0,700;1,400&family=Inter&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="brand">
+        <div class="logo">
+          <img src="/images/main_logo.png" alt="Companie's Logo" />
+        </div>
+        <div>
+          <p>Professional web development since 2010</p>
+        </div>
+      </div>
+    </header>
+    <section class="support">
+      <h1>Website support of creative agency Caveat</h1>
+      <div>
+        <p class="stack">
+          <b>Stack:</b>
```

Comment:
use `<strong>` here to improve SEO and screenreaders experience

Reply:
done